app/views/brexit_landing_page/show.html.erb

```diff
@@ -1,14 +1,14 @@
 <% content_for :page_class, "taxon-page taxon-page--grid" %>
-<% content_for :extra_headers do %>
-  <link rel="preload" as="image" href="<%= asset_path 'chevron-extend-hover.svg' %>">
+<% content_for :meta_section do %>
```

Comment:
perhaps this belongs in meta_tags, I don't really know why we need meta_tags and meta_section since they seem to serve similar purpose although the only thing we seem to have in meta_section is `<meta name="govuk:section" content="<%= meta_section %>">`

Reply:
I've altered all content_for's using meta_section to now use meta_tags, as you said there's no clear reason and they were originally all housed under one - 'extra_headers'. See here https://github.com/alphagov/collections/commit/031d6615893076cffcd44cc18420287ce99d45f0

Reply:
Great thanks Pete.

I noticed there's a couple of files where meta tags are defined twice:
https://github.com/alphagov/collections/blob/1e53550626aeb5e653e693a35643a69c2b1efc27/app/views/second_level_browse_page/show.html.erb#L3-L10

and

https://github.com/alphagov/collections/blob/1e53550626aeb5e653e693a35643a69c2b1efc27/app/views/subtopics/show.html.erb#L3-L21

Reply:
Now there is one per file! 🎉